main.qml

```diff
@@ -908,38 +890,25 @@ ApplicationWindow {
     function handleSweepUnmixable() {
         console.log("Creating transaction: ")
 
+        var sweepUnmixable = true;
+        txConfirmationPopup.open();
         transaction = currentWallet.createSweepUnmixableTransaction();
         if (transaction.status !== PendingTransaction.Status_Ok) {
             console.error("Can't create transaction: ", transaction.errorString);
-            informationPopup.title = qsTr("Error") + translationManager.emptyString;
-            informationPopup.text  = qsTr("Can't create transaction: ") + transaction.errorString
-            informationPopup.icon  = StandardIcon.Critical
-            informationPopup.onCloseCallback = null
-            informationPopup.open();
+            txConfirmationPopup.errorText.text  = qsTr("Can't create transaction: ") + transaction.errorString + translationManager.emptyString     
             // deleting transaction object, we don't want memleaks
             currentWallet.disposeTransaction(transaction);
 
         } else if (transaction.txCount == 0) {
-            informationPopup.title = qsTr("Error") + translationManager.emptyString
-            informationPopup.text  = qsTr("No unmixable outputs to sweep") + translationManager.emptyString
-            informationPopup.icon = StandardIcon.Information
-            informationPopup.onCloseCallback = null
-            informationPopup.open()
+            console.error("No unmixable outputs to sweep");
```

Comment:
I'm asking myself if wouldn't be better to use less technical terminology. "unmixable outputs" is probably uncomprehensible for most people. Same for the other occurencies.

Reply:
This is only available in the advanced options, which are only visible in wallets using advanced mode.

Reply:
I don't have a better phrasing to suggest, so feel free to ignore, but if we can find an alternative would be better. Using the advanced mode doesn't necessarely mean that the person using it has deep knowledge of how Monero works, and i feel this phrase will be confusing for both users and translators. AFAIR we always used it, i just think would be a good idea to change it :)

Reply:
"stale outputs"?